lib/util/constants.dart

```diff
@@ -37,6 +37,7 @@ const DEEP_LINKS = [
 ];
 const FF_ARTIST_COLLECTOR =
     'https://feralfile.com/docs/artist-collector-rights';
+const FF_EXHIBITION_PREFIX = "https://feralfile.com/exhibitions/";
```

Comment:
suggest don't using hardcode the link here, it should be configurable. We already have the function `String feralFileExhibitionUrl(String slug)`. I think you can reuse it.